src/usdb_syncer/gui/forms/SettingsDialog.ui

```diff
@@ -543,6 +553,41 @@
            </layout>
           </widget>
          </item>
+         <item>
+          <widget class="QGroupBox" name="groupBox_3">
+           <property name="minimumSize">
+            <size>
+             <width>0</width>
+             <height>80</height>
+            </size>
+           </property>
+           <property name="title">
+            <string>Discord integration for outdated resources</string>
+           </property>
+           <layout class="QGridLayout" name="gridLayout_11">
+            <item row="0" column="0">
+             <widget class="QLabel" name="label_discord">
+              <property name="text">
+               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Allow &lt;span style=&quot; font-style:italic;&quot;&gt;anonymous&lt;/span&gt; notifications:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+              </property>
+             </widget>
+            </item>
+            <item row="0" column="1">
+             <widget class="QCheckBox" name="checkBox_discord_allowed">
+              <property name="toolTip">
+               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This will send a notification to our Karaoke Discord channel and report unavailable resources so that they can be fixed. This reporting is completely &lt;span style=&quot; font-weight:700; font-style:italic;&quot;&gt;anonymous&lt;/span&gt;. Please keep this enabled to support the community.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+              </property>
+              <property name="text">
+               <string/>
+              </property>
+              <property name="checked">
+               <bool>true</bool>
```

Comment:
Would really prefer to have this opt-in. Surely, there would still be more than enough participants.